.github/workflows/deploy_stage.yml

```diff
@@ -38,12 +38,15 @@ jobs:
         env:
           AIO_RUNTIME_NAMESPACE: ${{ secrets.AIO_RUNTIME_NAMESPACE_STAGE }}
           AIO_RUNTIME_AUTH: ${{ secrets.AIO_RUNTIME_AUTH_STAGE }}
-          GROUP_CHECK_URL: ${{ vars.GROUP_CHECK_URL }}
-          ENABLE_PREVEW: ${{ vars.ENABLE_PREVEW }}
           BULK_PREVIEW_CHECK_INTERVAL: ${{ vars.BULK_PREVIEW_CHECK_INTERVAL }}
-          MAX_BULK_PREVIEW_CHECKS: ${{ vars.MAX_BULK_PREVIEW_CHECKS }}
+          CERT_THUMB_PRINT: ${{ secrets.CERT_THUMB_PRINT }}
+          CLIENT_ID: ${{ secrets.CLIENT_ID }}
+          ENABLE_PREVEW: ${{ vars.ENABLE_PREVEW }}
           GRAYBOX_USER_GROUPS: ${{ vars.GRAYBOX_USER_GROUPS }}
+          GROUP_CHECK_URL: ${{ vars.GROUP_CHECK_URL }}
+          MAX_BULK_PREVIEW_CHECKS: ${{ vars.MAX_BULK_PREVIEW_CHECKS }}
           SPLUNK_HEC__HEC_TOKEN: ${{ secrets.SPLUNK_HEC_TOKEN_STAGE }}
+          TENANT_ID: ${{ secrets.TENANT_ID }}
```

Comment:
@arshadparwaiz It is still missing a few I think. 
`app.config.yaml` file reads these values and maps to the variables.
https://github.com/adobecom/graybox-io/blob/main/app.config.yaml#L8-L21

Reply:
Those should be added then. I am assuming your .env file already has those values.

Reply:
Added the secrets in github and then updated the vars in deploy_stage.yaml

Reply:
Thanks Sunil